packages/components/src/components/carousel/carousel.scss

```diff
@@ -12,14 +16,22 @@
     }
     &__box {
         overflow: hidden;
+
+        &--mt5 {
+            margin-left: -2.4rem;
+        }
     }
     &__wrapper {
         display: flex;
         flex-direction: row;
         transition: transform 0.3s linear;
-        padding: 0.5rem 0;
         width: fit-content;
+        padding: 0.5rem 0;
         height: 100%;
+
+        &--mt5 {
+            padding: 0 0 0.5rem;
```

Comment:
Would just giving `padding-top: unset;`/ `padding-top: 0;` not work since that's the only value we want to change?